docs/sources/zendesk.md

```diff
@@ -0,0 +1,60 @@
+# Event source for Zendesk
+
+This event source registers itself as a notification receiver in Zendesk in order to capture events such as ticket
+creations.
+
+## Prerequisite: API token
+
+An API token is required in order to let the TriggerMesh Zendesk event source create a corresponding [Target][zd-target]
+and [Trigger][zd-trigger] in your Zendesk account. To create a new API token from the Zendesk Admin interface, follow
+the instructions at [Generating a new API token ][zd-token].
+
+## Deploying an instance of the Source
+
+Open the Bridge creation screen and add a source of type `Zendesk`.
+
+![Adding a Zendesk source](../images/zendesk-source/create-bridge-1.png)
+
+In the Source creation form, give a name to the event source and add the following information:
+
+* **Email**: Email address associated with the Zendesk account.
+* [**Subdomain**][zd-subdom]: Name of the Zendesk subdomain, without the `zendesk.com` domain or `https://` scheme.
+* [**Token**][zd-token]: API token to communicate with the Zendesk API, as described in the previous section.
+* **Webhook username/password**: arbitrary user name and password, used to verify event callbacks.
+
+![Zendesk source form](../images/zendesk-source/create-bridge-2.png)
+
+After clicking the `Save` button, you will be taken back to the Bridge editor. Proceed to adding the remaining
+components to the Bridge, then submit it.
+
+![Bridge overview](../images/zendesk-source/create-bridge-3.png)
+
+A ready status on the main _Bridges_ page indicates that the Zendesk [Target][zd-target] and [Trigger][zd-trigger] were
+successfully created and that the event source is ready to forward events from Zendesk.
+
+![Bridge status](../images/awseventbridge-target/create-bridge-4.png)
+
+To verify the successful deployment of the Zendesk event source, navigate to the [Targets][zd-target] tab of the
+_Extensions_ screen in the Zendesk Admin interface, below the _Settings_ section. The event source instance should have
+created a Target following the naming pattern `io.triggermesh.zendesksource.<triggermesh user>.<source name>`.
```

Comment:
io.triggermesh.zendesksource.<namespace>.<source name>.

Reply:
?! oh nooo!! markdown got the best of me lol 
*`io.triggermesh.zendesksource.<triggermesh user>.<source name>.`

Reply:
Should users be aware that their user name is a "namespace"? I believe not, this is a Kubernetes notion, not a TriggerMesh notion.

Reply:
Done in 6f9fc90